demo/sizing-and-spacing.html

```diff
@@ -244,6 +244,22 @@ <h3>Tall</h3>
       </template>
     </nice-demo-snippet>
 
+    <h2 demo-section>Compact preset</h2>
+    <p>The "compact" preset is a set of predefined sizing/spacing properties that reduce the overall application size to better fit a large number of content on the screen.</p>
```

Comment:
Clarify documentation and fix grammar:

“The compact preset defines values for the sizing and spacing properties to reduce the visual space required by components to better fit a large amount of content on the screen.”

Reply:
Done.